integration_test/pg/storage_test.exs

```diff
@@ -139,6 +139,19 @@ defmodule Ecto.Integration.StorageTest do
     def change, do: :ok
   end
 
+  defmodule CreateTestSchemaMigration do
+    use Ecto.Migration
+
+    def change do
+      execute TestRepo.create_prefix("test_schema")
+
+      create table(:schema_migrations, prefix: "test_schema") do
+        add :version, :integer
+        add :inserted_at, :timestamp
+      end
```

Comment:
```suggestion
```
I am pretty sure this should be taken care of when you run your first migration on the schema: https://github.com/elixir-ecto/ecto_sql/blob/master/lib/ecto/migration/schema_migration.ex#L24